custom-commands/experimental/custom-commands-offline/Run-CC-Offline.md

```diff
@@ -0,0 +1,123 @@
+# Running the Custom Command Embedded Service on Windows

```

Comment:
The title says Windows. In the body of the document it includes other OS including macos and Linux. Is the document meant to cover all of them?